packages/cojson/src/sync.ts

```diff
@@ -511,7 +480,10 @@ export class SyncManager {
       }
     }
 
-    if (entry.state.type === "unknown") {
+    // When we receive a known state from a peer and the header is "false"
+    // it means that the peer doesn't know anything about this coValue
+    // So we can say that the coValue has been not found on this peer
+    if (entry.state.type !== "available") {
       const availableOnPeer = peer.optimisticKnownStates.get(msg.id)?.header;
 
       if (!availableOnPeer) {
```

Comment:
If this message is lost we could end up infinitely waiting when loading a coValue.

This problem makes me think about @marinoska suggestion about having "an ack of the ack".
I think known messages need to have the insurance to be delivered because if we miss some of them some CoValues might get stuck.

Reply:
It's actually the same with the "load" messages. 

The only ones we can afford to lose are the "content" messages.